glacier_mapping/train.py

```diff
@@ -70,15 +73,18 @@ def validate(loader, frame, metrics_opts):
     :return (val_loss, metrics): A tuple containing the average validation loss
       and the metrics on the validation set.
     """
-    loss, metrics = 0, []
+    loss, metrics = 0, {}
     for x, y in loader:
+        channel_first = lambda x: x.permute(0, 3, 1, 2)
```

Comment:
This can go outside the for loop. It's a little confusing to have an x defined in the loop as well as in this lambda.